lisp/i18n/ts/fr_FR/lisp.ts

```diff
@@ -2321,7 +2291,7 @@
   <context>
     <name>QColorButton</name>
     <message>
-      <location filename="../../../ui/widgets/qcolorbutton.py" line="38"/>
+      <location filename="../../../ui/widgets/colorbutton.py" line="38"/>
```

Comment:
mmm well... @FrancescoCeruti it looks like a typo here. A translator shouldn't change the code, isn't?